content/cli/v6/commands/npm.md

```diff
@@ -33,7 +33,7 @@ Run `npm help` to get a list of available commands.
 
 npm is configured to use npm, Inc.'s public registry at
 https://registry.npmjs.org by default. Use of the npm public registry is
-subject to terms of use available at https://www.npmjs.com/policies/terms.
+subject to terms of use available at https://docs.npmjs.com/policies/terms.
```

Comment:
This needs to be updated in https://github.com/npm/cli/blob/v6/docs/content/cli-commands/npm.md